src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -63,11 +72,18 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
         // run FilterMutectCalls
         new Main().instanceMain(makeCommandLineArgs(Arrays.asList("-V", unfilteredVcf.getAbsolutePath(), "-O", filteredVcf.getAbsolutePath()), "FilterMutectCalls"));
 
-        final Pair<Double, Double> concordance = calculateConcordance(filteredVcf, truthVcf);
-        final double sensitivity = concordance.getLeft();
-        final double fdr = concordance.getRight();
-        Assert.assertTrue("Sensitivity " + sensitivity + " was lower than required sensitivity of " + requiredSensitivity, sensitivity > requiredSensitivity);
-        Assert.assertTrue(fdr < 0.5);
+
+        // run Concordance
+        final File concordanceSummary = createTempFile("concordance", ".txt");
+        new Main().instanceMain(makeCommandLineArgs(Arrays.asList("-truth", truthVcf.getAbsolutePath(), "-eval", unfilteredVcf.getAbsolutePath(), "-L", "20", "-XL", mask.getAbsolutePath(), "-summary", concordanceSummary.getAbsolutePath()), "Concordance"));
+
+        final List<ConcordanceSummaryRecord> summaryRecords = new ConcordanceSummaryRecord.Reader(concordanceSummary).toList();
+        summaryRecords.forEach(rec -> {
+            if (rec.getTruePositives() + rec.getFalseNegatives() > 0) {
```

Comment:
This is to make sure that the denominator of `getSensitivity()` wouldn't be zero.